packages/bot/src/utils/scratchHelper.js

```diff
@@ -0,0 +1,12 @@
+import config from '../constants';
```

Comment:
Could you put this in `packages/bot/src/scratch/utils.js`?

Reply:
sure. thanks